src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -105,6 +105,11 @@ export class IssuesDataTable extends DataSource<Issue> {
                     return issue.assignees.includes(this.assignee.login);
                   }
                 });
+              } else {
+                // If no assignee, filter the unassigned issues only.
+                data = data.filter((issue) => {
+                  return issue.assignees.length === 0;
```

Comment:
this should be `return issue.issueOrPr !== "PullRequest" && issue.assignees.length === 0;` to avoid the problem of PR appearing twice (refer to overall comment for the screenshot)

Reply:
thanks for the catch, just added a commit adding your change :)